.travis.yml

```diff
@@ -9,12 +9,17 @@ python:
   - "2.7"
   - "3.5"
 
+env:
+  - TEST_DIR=root TEST_TYPE=lint
+  - TEST_DIR=root TEST_TYPE=unit
+  - TEST_DIR=utils
+
 install:
   - pip install -r requirements.txt
   - pip install tox-travis
 
 script:
   # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
   - ansible-playbook --syntax-check playbooks/byo/config.yml
-  - tox
-  - cd utils && tox
+  - if [[ "${TEST_DIR}" == "root" ]]; then tox; fi
+  - if [[ "${TEST_DIR}" == "utils" ]]; then pushd utils; tox; popd; fi
```

Comment:
Left over from debugging where I was trying to understand the travis cache size, cleaning up.